Tests/TuistLoaderTests/Loaders/SwiftPackageManagerGraphLoaderTests.swift

```diff
@@ -0,0 +1,108 @@
+import MockableTest
+import TuistCore
+import TuistSupport
+import XCTest
+
+@testable import TuistLoader
+@testable import TuistSupportTesting
```

Comment:
```suggestion
import TuistSupportTesting
```

A basic import should be sufficient. We've been doing this in the past, but we're trying to use `@testable` only for modules that we're actually testing, everything else should be "normal" imports